### PR TITLE
chore: update node version

### LINKED ISSUE #2275: Update to Node v16
Following the discussion in https://github.com/Qiskit/qiskit.org/discussions/2269, we need to upgrade to Node v16.

### Tasks:

- [x] Update Qiskit-Vue - https://github.com/qiskit-community/qiskit-vue/pull/36
- [x] Update Qiskit-Web-Components - https://github.com/qiskit-community/qiskit-web-components/pull/29
- [ ] Update Qiskit.org
- [x] Update Platypus - https://github.com/Qiskit/platypus/pull/1783

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Fixes #2275 

Co-author: @eddybrando, @y4izus 

## Implementation details

- updates node v to 16
- updated node version w/in the deploy and verify ymls, along w/ the dockerfile

### CODE REVIEW

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you Randy!! Just one request: I think we should also update the [Dockerfile](https://github.com/Qiskit/qiskit.org/blob/main/Dockerfile#L1)

On the other hand, I didn't realized we need node 16 because we are using the old banner:
Captura de pantalla 2023-01-16 a las 13 19 19
 so we should first [update the banner](https://github.com/Qiskit/qiskit.org/issues/2360) before upgrade to node 18

**Comment by eddybrando** [comment]
@techtolentino, I heard you like PRs, so I created a PR in your PR: https://github.com/Qiskit/qiskit.org/pull/2919

Please ignore the name of the branch. Initially I thought upgrading to Node v18 would be easier. But this PR at least solves the issues with other Node 16 versions.

If merging it causes issues (package-lock merges can be messy), we can replicate the changes I made in your PR if you prefer.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: "14"
+          node-version: "16.10.0"
```
Comment (eddybrando) [inline .github/workflows/deploy.yml:26]: https://github.com/Qiskit/qiskit.org/pull/2919 makes now possible to set all 16 versions without the need to specify 16.10.

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ on:
   pull_request:
 
 env:
-  NODE_VERSION: "14"
+  NODE_VERSION: "16.10.0"
```
Comment (eddybrando) [inline .github/workflows/verify.yml:8]: https://github.com/Qiskit/qiskit.org/pull/2919 makes now possible to set all 16 versions without the need to specify 16.10.

**Review by eddybrando — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (techtolentino) [reply .github/workflows/deploy.yml]: Thanks @eddybrando - resolved w/ https://github.com/Qiskit/qiskit.org/pull/2912/commits/c085b6c1fd6799610247849abee843554c6fe9a8

File: .github/workflows/verify.yml (hunk 2 above)
Comment (techtolentino) [reply .github/workflows/verify.yml]: Resolved w/ https://github.com/Qiskit/qiskit.org/pull/2912/commits/c085b6c1fd6799610247849abee843554c6fe9a8

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM node:14-alpine
+FROM node:16
```
Comment (y4izus) [inline Dockerfile:1]: I would prefer to keep using the `alpine` image. More context about the [image variants](https://hub.docker.com/_/node#:~:text=to%2Ddate%20documentation.-,Image%20Variants,-The%20node%20images) for node on Dockerfile

```suggestion
FROM node:16-alpine
```

**Review by y4izus — CHANGES_REQUESTED** [review]
Just one more small change and we finish Randy!! thank you!

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you again!! the last change!!

**Review by y4izus — APPROVED** [review]
LGTM!! Thank you very much!

**Pull request merged**